src/diffusers/pipelines/stable_diffusion/pipeline_flax_stable_diffusion.py

```diff
@@ -400,13 +400,13 @@ def __call__(
     static_broadcasted_argnums=(0, 4, 5, 6),
 )
 def _p_generate(
-    pipe,
+    pipe,  # static
```

Comment:
```suggestion
    pipe,
```

Can we please remove the line comments here?